resources/views/profile/update-password-form.blade.php

```diff
@@ -1,4 +1,4 @@
-<div x-data="{isTyping: false}">
+<div>
```

Comment:
you deleted it here but you still use `isTyping` on the registration form :bleh: does it have to change everywhere? I thought design showed the default text on the register page until you start typing?

Reply:
In relation to this, I don't particularly mind if this it triggered when the user starts typing or if we show it by default. Within the profile settings I thought it would make sense to show it straight away, but you are correct @ItsANameToo that the Registration designs do not shows the password checklist by default. So, I guess in the interest of consistency we should make the password checklist show as the user is typing across the whole of Nodem including both Registration and Security Settings pages.

Reply:
@Highjhacker see above